resources/web/wwi/x3d_scene.js

```diff
@@ -86,6 +86,12 @@ class X3dScene { // eslint-disable-line no-unused-vars
   }
 
   render() {
+    if (this.nextRenderingTime && this.nextRenderingTime > (new Date()).getTime()) {
+      if (!this.renderingTimeout)
+        this.renderingTimeout = setTimeout(() => this.render(), 40);
```

Comment:
I was wondering if we shouldn't use the exact delay here instead of 40, e.g.
```
const dt = this.nextRenderingTime - (new Date()).getTime();
```

Reply:
Yes, I can change it.
But the timeout of 40 is so small that I doubt there is a visual difference.

Reply:
I agree, there won't probably be any visual difference, but it looks better to me to use the actual value of the delay rather than assuming it's 40 ms. Do you agree?

Reply:
yes